ogusa/parameters.py

```diff
@@ -137,7 +137,7 @@ def get_parameters_from_file():
 
 
 def get_parameters(test=False, baseline=False, guid='', user_modifiable=False, metadata=False,
-                   tx_func_est_path=None):
+                   tx_func_est_path=None, start_year=2018):
     '''
```

Comment:
@jdebacker `start_year` is added as a keyword argument where `get_parameters` is called in `execute.py`.  Is this what you meant?

Reply:
@hdoupe Yes - so line 59 of `execute.py` would be:

```
run_params = ogusa.parameters.get_parameters(
        test=test, start_year=start_year=user_params['start_year'], baseline=baseline, guid=guid,
        tx_func_est_path=os.path.join(
            output_base,'TxFuncEst_{}.pkl'.format(guid)
```